### PR TITLE
Run more CFE instances for during testing

### PR DESCRIPTION
**Pull request by mraleph**

### CODE REVIEW

File: packages/flutter_tools/lib/src/test/flutter_platform.dart [hunk 1]
```diff
@@ -97,9 +99,11 @@ class _CompilationRequest {
   _CompilationRequest(this.path, this.result);
 }
 
-// This class is a wrapper around compiler that allows multiple isolates to
+// This class is a wrapper around compiler that allows multiple tests to
 // enqueue compilation requests, but ensures only one compilation at a time.
```
Comment (gspencergoog) [inline packages/flutter_tools/lib/src/test/flutter_platform.dart]: Is it still just one compilation at a time?

**Comment by jensjoha** [comment]
I have an alternative that just launches one server per simultaneous test (i.e. whatever controls how many test to run at once controls the number of servers; i.e. no `Platform.numberOfProcessors ~/ 2`). The code, frankly is much simpler, and on my machine at least it works fine... I will upload shortly, and someone can decide...

**Comment by Hixie** [comment]
What's the status of this PR? Are we still interested in going down this path?

**Comment by mraleph** [comment]
@Hixie I think no for now. We have a much better win in the pipeline on the Dart side: https://dart-review.googlesource.com/c/47020/

**Pull request closed without merging**